packages/react-native-web/src/exports/Dimensions/index.js

```diff
@@ -77,13 +77,31 @@ export default class Dimensions {
     }
 
     const win = window;
-    const docEl = win.document.documentElement;
+    let height;
+    let width;
+
+    /*
+     * iOS does not update viewport dimensions on keyboard open/close
+     * So, we are using window.visualViewport(https://developer.mozilla.org/en-US/docs/Web/API/VisualViewport)
+     * instead of document.documentElement.clientHeight
+     * But we are still using document.documentElement.clientWidth as fallback
+     * because it was requested by the original repo owner
```

Comment:
NAB: I think its fine to be kept like this
```suggestion
     * document.documentElement.clientWidth is used as a fallback
```